src/main/java/edu/umich/tl/ApiCallHandler.java

```diff
@@ -112,7 +187,7 @@ private HttpResponse apiDirectCanvas(String url) {
 		return httpResponse;
```

Comment:
It's not clear from the pull request.  Is it documented that the httpResponse may be null?